### DISCUSSION TITLE
Fabric YouTube Playlist

### DISCUSSION THREAD

**Discussion by CPAtoCybersecurity** [danielmiessler/Fabric#123] [Show and tell]
Here’s my YouTube playlist with some demos to help “build, continuously improve and encourage the adoption of fabric.”  As you can see I’m brand new to coding but I hope to help other rookies cross over and look forward to learning in this community. Questions and feedback are welcome.
https://youtube.com/playlist?list=PL_PGKRbFLsda6ZIUS9UCAlFU107O4mBuY&si=vx-Lv24CVDbsFAMq

**Comment by danielmiessler** [comment]
Amazing!

**Comment by CPAtoCybersecurity** [comment]
I’ve added a video to the playlist on using the new context switch. This required installing the latest fabric code where I ran into a snag with QuickStart instruction #6 that I overcame talking to Chat GPT as demoed in the video. Feedback welcome.
